.github/workflows/github_actions.yml

```diff
@@ -0,0 +1,102 @@
+name: TileDB-Trino
+
+on:
+  push:
+    tags:
+      - '*'
+    branches:
+      - 'master'
+  pull_request:
+
+jobs:
+  Test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-18.04, macos-10.15 ]
+
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          unset SYSTEM
+          set -e pipefail
+          open /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg
+          sudo installer -pkg /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg -allowUntrusted -target /
+        shell: bash
+        name: Install_system_headers_mac_only
+        if: matrix.os == 'macOS-10.14'
+
+      - run: |
+          unset SYSTEM
+          export JAVA_HOME=$JAVA_HOME_11_X64
+          mvn checkstyle:checkstyle
+        shell: bash
+        name: checkStyle
+
+      - run: |
+          unset SYSTEM
+          export JAVA_HOME=$JAVA_HOME_11_X64
+          mvn clean package -DskipTests
+        shell: bash
+        name: checkFormat
+
+      - run: |
```

Comment:
These could all be combined into a single `run: |` block, but make sure to add `set -eu` at the top so an individual failure will fail the step (assuming GHA has the same default as AZP).

Reply:
Yes, but I chose to have the steps separately for better clarity when running the CI. Let me know what you think.

Reply:
Ok no problem